<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="priceSpinBox">
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="costSpinBox">
       <property name="maximum">
        <double>9999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cost:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Quantity:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="quantitySpinBox">
       <property name="maximum">
        <double>99999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Un. of Measure.:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Barcode:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="barcodeLineEdit"/>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="unitComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="newUnitMeasureBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>112</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>unitComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>setUnitMeasure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>descriptionLineEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newUnitMeasureBtn</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>newUnitMeasure()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setUnitMeasure()</slot>
  <slot>save()</slot>
  <slot>cancel()</slot>
  <slot>newUnitMeasure()</slot>
 </slots>
</ui>
